src/XMakeBuildEngine/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -61,14 +58,14 @@ protected override ICollection<I> SelectItems(ImmutableList<ItemData>.Builder li
                     }
                 }
 
-                foreach (var operation in _operations)
+                foreach (var fragment in _itemSpec.Fragments)
                 {
-                    if (operation.Item1 == ItemOperationType.Expression)
+                    if (fragment is ItemExpressionFragment<P, I>)
```

Comment:
Sounds like a good plan.

---
In reply to: [75950700](https://github.com/Microsoft/msbuild/pull/934#discussion_r75950700) [](ancestors = 75950700)